EIPS/eip-1895.md

```diff
@@ -0,0 +1,166 @@
+---
+eip: 1895
+title: Support for an Elliptic Curve Cycles
+author: Alexandre Belling <alexandrebelling8@gmail.com>
+discussions-to: https://ethresear.ch/t/reducing-the-verification-cost-of-a-snark-through-hierarchical-aggregation/5128/7
```

Comment:
This link doesn't go to the top of start of the discussion, but to one particular comment by you.